connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -406,7 +406,6 @@ public void testReplication() throws Exception {
                     "New topic was not replicated to primary cluster.");
```

Comment:
If we don't add new integration testing coverage for this PR, we should also revert any changes made to this file.